src/med_bench/nuisances/density.py

```diff
@@ -0,0 +1,315 @@
+"""
+the objective of this script is to provide nuisance estimators 
+for mediation in causal inference
+"""
+
+import numpy as np
+
+from sklearn.base import clone
+
+from med_bench.utils.utils import _get_train_test_lists, _get_interactions
+from sklearn.neighbors import NearestNeighbors, KernelDensity
+from sklearn.base import BaseEstimator
+from joblib import Parallel, delayed
+
+def estimate_mediator_density(t, m, x, y, crossfit, clf_m,
+                                        interaction, fit=False):
+    """
+    Estimate mediator density f(M|T,X)
+    with train test lists from crossfitting
+
+    Returns
+    -------
+    f_t0: list
+        contains array-like, shape (n_samples) probabilities f(M=m|T=0,X)
+    f_t1, list
+        contains array-like, shape (n_samples) probabilities f(M=m|T=1,X)
+    f_m0x, array-like, shape (n_samples)
+        probabilities f(M|T=0,X)
+    f_m1x, array-like, shape (n_samples)
+        probabilities f(M|T=1,X)
+    """
+    # if not is_array_integer(m):
```

Comment:
remove unused lines

Reply:
This is addressed in commit 6378d4318af2c23ec76999d758ad47475dd26d6e